apps/cms-v2/src/modules/providers/payloadcms/payloadcms-webhooks-processor.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  WebhookProductFragment,
+  WebhookProductVariantFragment,
+} from "../../../../generated/graphql";
+
+import { PayloadCmsProviderConfig } from "@/modules/configuration/schemas/payloadcms-provider.schema";
+import { createLogger } from "@saleor/apps-shared";
+import { ProductWebhooksProcessor } from "../../webhooks-operations/product-webhooks-processor";
+import { PayloadCMSClient } from "./payloadcms-client";
+
+/*
+ * todo error handling
+ */
+export class PayloadCmsWebhooksProcessor implements ProductWebhooksProcessor {
+  // private client: DatoCMSClient;
+  private logger = createLogger({ name: "PayloadCmsWebhooksProcessor" });
+
+  constructor(private providerConfig: PayloadCmsProviderConfig.FullShape) {}
+
+  async onProductVariantUpdated(productVariant: WebhookProductVariantFragment): Promise<void> {
+    this.logger.trace("onProductVariantUpdated called");
+
+    const client = new PayloadCMSClient();
```

Comment:
**suggestion (non-blocking)**:
I think you can initialize the client in the constructor rather than in each method.